packages/wxt/src/@types/globals.d.ts

```diff
@@ -1,6 +1,4 @@
-declare const __DEV_SERVER_PROTOCOL__: string;
-declare const __DEV_SERVER_HOSTNAME__: string;
-declare const __DEV_SERVER_PORT__: string;
+declare const __DEV_SERVER_ORIGIN__: string;
```

Comment:
Instead of constructing a best-guess URL to use to contact the dev server, we can use `ORIGIN` here